dotnet/Directory.Build.props

```diff
@@ -13,6 +13,10 @@
     <CSNoWarn>CS1998;CS1591;CS8002;</CSNoWarn>
     <SKEXPNoWarn>SKEXP0001;SKEXP0010;SKEXP0020</SKEXPNoWarn>
     <NoWarn>$(NoWarn);$(CSNoWarn);$(SKEXPNoWarn);NU5104</NoWarn>
+
+    <!-- Some properties require a different version of System.Json.Text > 8.0.5
+    Some csprojs override the version and this is a workaround from duplicate package references -->
```

Comment:
Maybe try simply removing the S.T.J package reference in `Directory.Build.props` instead? We probably don't need that global reference after updating M.E.A.I version